webrender/src/frame.rs

```diff
@@ -340,12 +340,12 @@ impl Frame {
         let layer_rect = LayerRect::new(LayerPoint::zero(),
                                         LayerSize::new(content_size.width + clip.origin.x,
                                                        content_size.height + clip.origin.y));
-        context.builder.push_layer(layer_rect,
-                                   &ClipRegion::simple(&layer_rect),
-                                   LayerToScrollTransform::identity(),
-                                   pipeline_id,
-                                   current_scroll_layer_id,
-                                   &[]);
+        context.builder.push_stacking_context(layer_rect,
+                                              &ClipRegion::simple(&layer_rect),
+                                              LayerToScrollTransform::identity(),
```

Comment:
Yep, I'd say let's not have the transformations in there. Since they only affect the *initial* placement of items, we should use it and forget about them, only keeping the reference frames in our sight.